.github/workflows/generate-drought-map.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate Drought Map
+
+on:
+  schedule:
+    - cron: 0 13 * * 4
```

Comment:
Is this safe to run on a timer? Will the python script fail safely without writing any updates if something unexpected happens?

Reply:
I think so. There are several checks in the Python script that will exit if our map is already current, or if the USDM file is not found. I'd really like to get @jbum's opinion though, just to be sure.

Reply:
(I am assuming @jbum wrote the original script here. Apologies if I'm wrong here.)

Reply:
He did :-)